crypto/src/test/java/org/hyperledger/besu/crypto/SignatureAlgorithmFactoryTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *
+ *  * Copyright ConsenSys AG.
```

Comment:
I think you've got too many `*` in each line :)

Reply:
I've fixed it